pkg/rolesformatter/yamlFormatter_test.go

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package rolesformatter
+
+import (
+	"testing"
+
+	"github.com/galasa-dev/cli/pkg/galasaapi"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestRolesYamlFormatterHasCorrectName(t *testing.T) {
+	formatter := NewRolesYamlFormatter()
+	assert.Equal(t, formatter.GetName(), "yaml")
+}
+
+func TestRolesYamlFormatterValidData(t *testing.T) {
+	// Given...
+	formatter := NewRolesYamlFormatter()
+	roles := createTestRoles()
+
+	// When...
+	actualFormattedOutput, err := formatter.FormatRoles(roles)
+
+	// Then...
+	assert.Nil(t, err)
+	expectedFormattedOutput := `apiversion: galasa-dev/v1alpha1
+kind: GalasaRole
+metadata:
+    id: role1Id
+    name: role1Name
+    description: role1Description
+    url: http://myHost/api/rbac/roles/role1Id
+data:
+    actions:
+        - action1
+        - action2
+---
+apiversion: galasa-dev/v1alpha1
```

Comment:
Should this be outputting `apiVersion` instead of `apiversion`?

Reply:
This is an odd one. The service is returning it, but the generated bean looks OK to me.
Perhaps we need to add the annotations for how the name should render to json explicitly.